### ISSUE TITLE
Bindable changed handlers are not called if VM implements `bind`.

### ISSUE THREAD

**Issue by JoshMcCullough** [aurelia/binding#372]
Handlers for bindable properties are not fired (the first time) when the VM also implements the `bind` pipeline step.

This will alert every time the `test` property changes:

```
export class MyClass {
    @bindable test

    testChanged() { alert('test'); }
}
```

This will alert every time the `test` property changes _except for the initial change_:

```
export class MyClass {
    @bindable test

    bind() { ... }

    testChanged() { alert('test'); }
}
```

This took me an hour to track down, finally @z-brooks [informed me of this rule](https://gitter.im/Aurelia/Discuss?at=570400588eec5313252e5384) and I was able to move forward. IMO this is behavior is very unclear.

**Comment by EisenbergEffect** [comment]
This enables bind to be called synchronously when a component is first bound, enabling startup binding code to reliably respond when all properties have been set by the binding system, but then to update asynchronously later on when they update. We'd love for you to send a PR to improve the documentation around this.

**Comment by JoshMcCullough** [comment]
I'm honestly not sure where this even is referenced in the docs (couldn't find it). If you point me to it, I'll be happy to update it if it is, indeed, unclear. Thanks!

**Comment by EisenbergEffect** [comment]
Perhaps a note in this section of the cheat sheet: http://aurelia.io/docs.html#/aurelia/framework/1.0.0-beta.1.1.4/doc/article/cheat-sheet/2
The src for that doc is in doc folder of the framework repo.

**Comment by JoshMcCullough** [comment]
Done: https://github.com/aurelia/framework/pull/390/commits/c2a85c93321a21618dbafa66f7c8de46098fa485

It ended up a bit wordy, but I think it's a relatively difficult thing to explain in a clear and concise manner.

**Comment by MaartenThomassen** [comment]
Is there any way to manually enable this feature? (or perhaps use an attibute on the viewmodel of the control). 

I currently have a control which needs the context which is gotten by implementing bind. But it also has a load of other properties for which (i assumed) the chane handler is called when the control is created. It leads to alot of error-prone code if i need to add a manual call to the change handler of each property in the bind call (especially when others in my team add properties and they forget about adding a call to it in the bind function).
